test/jerboa/format/body/attribute/lifetime_test.exs

```diff
@@ -0,0 +1,43 @@
+defmodule Jerboa.Format.Body.Attribute.LifetimeTest do
+  use ExUnit.Case
+  use Quixir
+
+  alias Jerboa.Format.Body.Attribute.Lifetime
+  alias Jerboa.Format.Lifetime.LengthError
+
+  @max_duration 2 |> :math.pow(32) |> :erlang.trunc() |> Kernel.-(1)
+
+  describe "decode/1" do
+    test "LIFETIME attribute with valid length" do
+      ptest duration: int(min: 0, max: @max_duration) do
+        lifetime = <<duration::32>>
+
+        assert {:ok, %Lifetime{duration: duration}} == Lifetime.decode(lifetime)
+      end
+    end
+
+    test "LIFETIME attribute with invalid length" do
+      ptest lifetime: string(min: 5, chars: :ascii) do
```

Comment:
Unfortunately Pollution (a library for generating properties) does not support generating binaries, so here I am generating an ASCII string, where each byte represents one character (now that I think about it ASCII modifier is probably not necessary here).